src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState, useEffect} from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { Main, Repo } from './pages'
+// import { UserInfo } from './components';
+import axios from 'axios';
+
+
+function App() {
+    const [ username, setUsername ] = useState();
+    const [repos, setRepos] = useState();
+    const [selectedRepo, setSelectedRepo] = useState()
+
+    useEffect(async () => {
+        try {
+            let {data} = await axios.get(`https://api.github.com/users/${username}/repos`)
+            setRepos(data)
+        } catch (err) {
+            console.warn(err);
```

Comment:
Maybe the website should show an error user not found message when searching for an incorrect username